bodhi/server/tasks/approve_testing.py

```diff
@@ -78,14 +78,18 @@ def approve_update(update: Update, db: Session):
     log.info(f'{update.alias} now meets testing requirements')
     # Only send email notification about the update reaching
     # testing approval on releases composed by bodhi
-    update.comment(
-        db,
-        str(config.get('testing_approval_msg')),
-        author='bodhi',
-        email_notification=update.release.composed_by_bodhi
-    )
-    notifications.publish(update_schemas.UpdateRequirementsMetStableV1.from_dict(
-        dict(update=update)))
+    if not update.autotime or update.karma >= update.stable_karma:
```

Comment:
So I am not a big fan of having the check for karma here since this adding yet another place where we check for it. Ideally we should only have one place where we check if the update meet the requirements the `meets_testing_requirements` method. 

Maybe we could add the comment in that method instead of here, but we need to check where this method is called then.

Reply:
Right. The comment is already in `meets_testing_requirements`